lib/packages_manager.py

```diff
@@ -25,7 +25,14 @@
 
 class PackagesManager(object):
     def __init__(self, packages_names):
-        self.packages_names = packages_names
+        """
+        Class that manages actions on packages.
+
+        Args:
+            packages_names: names of the packages that will be processed.
+                Leave empty to discover all available packages.
+        """
+        self.packages_names = packages_names or config.discover_packages()
```

Comment:
This was needed for a previous version of the PR that had a slightly different functionality. I am dropping this patch.